src/url.ts

```diff
@@ -1,9 +1,10 @@
-import url from 'url';
+// tslint:disable-next-line no-require-imports
+const {parse} = require('url');
```

Comment:
Can you just do this in the branch where we find window is undefined? I think this might blow up otherwise.